docs/sources/explore/_index.md

```diff
@@ -59,6 +59,46 @@ In split view, timepickers for both panels can be linked (if you change one, the
 
 To close the newly created query, click on the Close Split button.
 
+## Share Explore URLs
+
+When using Explore, the URL in the browser address bar is updated as you make changes to the queries. This allows you to share the URLs with others or bookmark them for later use.
+
+### Generating Explore URLs from external tools
+
+Because Explore URLs have a defined structure, you can build an URL from external tools and open it in Grafana. The URL structure is:
```

Comment:
```suggestion
Because Explore URLs have a defined structure, you can build a URL from external tools and open it in Grafana. The URL structure is:
```